### PR TITLE
Improve FT2Font docstrings.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Summary

(See https://github.com/matplotlib/matplotlib/pull/15009 for the format.)

I know it's still not perfect, but let's do things incrementally...

## PR Checklist

- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
- [ ] New features are documented, with examples if plot related.
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] Conforms to Matplotlib style conventions (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).

### CODE REVIEW

File: src/ft2font_wrapper.cpp [hunk 1]
```diff
@@ -507,11 +514,13 @@ static PyObject *PyFT2Font_get_kerning(PyFT2Font *self, PyObject *args, PyObject
...
 
 const char *PyFT2Font_set_text__doc__ =
-    "set_text(s, angle)\n"
-    "\n"
-    "Set the text string and angle.\n"
-    "You must call this before draw_glyphs_to_bitmap\n"
-    "A sequence of x,y positions is returned";
+    "set_text(string, angle, flags)\n"
+    "--\n\n"
+    "Set the text *string and *angle*.\n"
```
Comment (QuLogic) [inline src/ft2font_wrapper.cpp:519]: Missing closing `*`.

File: src/ft2font_wrapper.cpp [hunk 2]
```diff
@@ -575,19 +584,20 @@ static PyObject *PyFT2Font_set_text(PyFT2Font *self, PyObject *args, PyObject *k
...
 {
     return PyLong_FromLong(self->x->get_num_glyphs());
 }
 
 const char *PyFT2Font_load_char__doc__ =
-    "load_char(charcode, flags=LOAD_FORCE_AUTOHINT)\n"
-    "\n"
-    "Load character with charcode in current fontfile and set glyph.\n"
-    "The flags argument can be a bitwise-or of the LOAD_XXX constants.\n"
+    "load_char(charcode, flags=32)\n"
```
Comment (QuLogic) [inline src/ft2font_wrapper.cpp:596]: Why is this one 32 now, when `load_text` doesn't show a default in the signature at all?

**Review by QuLogic — APPROVED** [review]

File: src/ft2font_wrapper.cpp (hunk 1 above)
Comment (anntzer) [reply src/ft2font_wrapper.cpp:519]: yes

File: src/ft2font_wrapper.cpp (hunk 2 above)
Comment (anntzer) [reply src/ft2font_wrapper.cpp:596]: because I had missed that

**Pull request merged**